content/about/notes/1.0/index.md

```diff
@@ -27,3 +27,7 @@ to usual pile of bug fixes and performance improvements.
   and mixer-policy components do not connect to the Kubernetes API endpoints of any of the remote
   clusters.  This results in a loss of telemetry fidelity as some of the metadata associated
   with workloads on remote clusters is incomplete.
+
+- There are Kubernetes manifests available for using Citadel standalone or with Citadel healthchecking enabled.
+  There is not a Helm implementation of these modes.  See [Issue 6922](https://github.com/istio/istio/issues/6922)
+  for more details.
```

Comment:
with this limitation, what do we recommend to folks for istio install?  We have been recommending template (helm) based.  cc @costinm

Reply:
@linsun I don't have a great answer other than to get it done in 1.1.  It is a LOT of work to implement standalone citadel.  the healthchecking work could be done pretty straightforwardly I suspect without much work.